docs/finality-provider-operation.md

```diff
@@ -731,7 +731,136 @@ The response includes:
+* `--keyring-backend`: The keyring backend to use.
+* `--home`: The home directory for the finality provider.
+* `--fees`: The fees to pay for the transaction, should be over `400ubbn`.
+  These fees are paid from the account specified in `--from`.
+
+Again, this command should ask to
+`confirm transaction before signing and broadcasting [y/N]:` and output the
+transaction hash.
+
+This will withdraw **ALL** accumulated rewards to the address you set in the
+`set-withdraw-addr` command if you set one. If no withdrawal address was set,
+the rewards will be withdrawn to your finality provider address.
+
+#### 6.2.3 Set Withdraw Address
+
+To set the withdraw address to the beneficiary key, use the following command:
+
+```shell
+fpd set-withdraw-addr <beneficiary-address> --from <registered-bbn-address>
+--keyring-backend test --home <home-dir> --fees <fees>
+```
+
+Parameters:
+
+* `<beneficiary-address>`: Corresponds to the beneficiary key and is where
+  withdraw rewards are sent to.
+* `<registered-bbn-address>`: Corresponds to the key used in registration and is where
+  withdraw rewards are sent to by default if no other address is set via `set-withdraw-addr`
+* `--from`: The finality provider's registered Babylon address.
+* `--keyring-backend`: The keyring backend to use.
+* `--home`: The home directory for the finality provider.
+* `--fees`: The fees to pay for the transaction, should be over `400ubbn`.
+  These fees are paid from the account specified in `--from`.
+
+This command should ask to
+`confirm transaction before signing and broadcasting [y/N]:` and output the
+transaction hash.
+
+### 6.2.4 Refunding finality provider
+
+To support the gas costs associated with committing randomness, which are not
+refunded by the protocol, we recommend setting up a refunding flow.
+This involves periodically transferring rewards from a beneficiary key to the
+operational key used by the finality provider.
+
+#### How to set up the refunding process
+
+1. Ensure you have two keys:
+   * Beneficiary Key: Receives staking rewards.
+   * Operational Key: Used by the finality provider daemon to submit transactions.
+
+  If this step hasnt been completed, follow the steps in
+  [5.2 Add key for the Babylon account](#52-add-key-for-the-babylon-account) and
+  create 2 additional keys.
+
+2. **Configure Withdrawals**:
+  Ensure the withdraw address is set to the beneficiary key using the
+  `set-withdraw-addr` command.
+
+  This sets the withdraw address to the beneficiary key to enable
+  withdrawing rewards periodically.
+
+3. **Fund the Operational Key**:
+  Set the operational key name in the keyring home directory in the
+  `[babylon]` config in `fpd.conf`.
+
+4. **Add a cron job**:
+  Add a cron job to transfer funds from the beneficiary key to the operational
+  key as needed.
+
+Only maintain the minimum balance required for finality provider operations in
+the operational key. Excess funds should be kept in more secure storage.
```

Comment:
```suggestion
Only maintain the minimum balance required for finality provider operations in
the operational key as this is a hot key. Excess funds should be kept safely in the benefiary address.
```

We'd better add some statistics here in case people ask how much funds should be reserved for operations. For reference:
```
in general committing randomnes should have basically constant cost and it relatievely cheak https://testnet.babylon.explorers.guru/transaction/FFFFE4E56DA3C5062D443A632483795F5F308CC5730DC6BF5FAF8421091450BF?height=687895. Given that it costs 0.000130BBN per commit, it seems something between 5-10bbn for operations should be enough for a long time
```